server/src/test/proto/spine/test/clientservice/customer/customer.proto

```diff
@@ -46,4 +46,7 @@ message Customer {
 
     // The name of the customer.
     people.PersonName name = 2;
+
+    // Info about customer's family.
+    repeated people.PersonName familyInfo = 3;
```

Comment:
It is not `familyInfo` either.

If you need something plural here, let's call it `nicknames`.